### DISCUSSION TITLE
Differences between OpenFisca US and Tax-Calculator for an EITC reform

### DISCUSSION THREAD

**Discussion by MaxGhenis** [PolicyEngine/policyengine-us#1047] [General]
A user is interested in the impact of repealing age limits on the childless EITC. OpenFisca US (via PolicyEngine) estimates that this would cost [$1.67 billion](https://policyengine.org/us/population-impact?eitc_min_age_childless=0&eitc_max_age_childless=100), while Tax-Calculator (via TaxBrain) estimates it would cost $3.49 billion in 2022, based on either the [CPS](https://compute.studio/PSLmodels/Tax-Brain/53287) or [PUF](https://compute.studio/PSLmodels/Tax-Brain/53288). We expect some differences, e.g. since we use the raw CPS from 2020 and policy for 2022, but this is larger than I'd expect.

Could it be due to different tax unit creation procedures? Has anyone else scored this reform? @nikhilwoodruff @MattHJensen @andersonfrailey

**Comment by nikhilwoodruff** [comment]
Can we assume that Tax-Calculator is using the CPS, since it's the same for both?

**Reply by MaxGhenis** [reply]
Not sure, filed https://github.com/PSLmodels/Tax-Brain/issues/190
